### ISSUE TITLE
Wrong IE path or missing file in v6.0.0

### ISSUE THREAD

**Issue by jlchereau** [vvo/selenium-standalone#261]
https://selenium-release.storage.googleapis.com/3.0/IEDriverServer_x64_3.0.1.zip does not seem to be available.

```
D:\WebStorm\Project>selenium-standalone install
----------
selenium-standalone installation starting
----------

---
selenium install:
from: https://selenium-release.storage.googleapis.com/3.0/selenium-server-standalone-3.0.1.jar
to: C:\Users\xxxxx\AppData\Roaming\npm\node_modules\selenium-standalone\.selenium\selenium-server\3.0.1-server.jar
---
chrome install:
from: https://chromedriver.storage.googleapis.com/2.27/chromedriver_win32.zip
to: C:\Users\xxxxx\AppData\Roaming\npm\node_modules\selenium-standalone\.selenium\chromedriver\2.27-x64-chromedriver
---
ie install:
from: https://selenium-release.storage.googleapis.com/3.0/IEDriverServer_x64_3.0.1.zip
to: C:\Users\xxxxx\AppData\Roaming\npm\node_modules\selenium-standalone\.selenium\iedriver\3.0.1-x64-IEDriverServer.exe
---
firefox install:
from: https://github.com/mozilla/geckodriver/releases/download/v0.13.0/geckodriver-v0.13.0-win64.zip
to: C:\Users\xxxxx\AppData\Roaming\npm\node_modules\selenium-standalone\.selenium\geckodriver\0.13.0-x64-geckodriver
C:\Users\xxxxx\AppData\Roaming\npm\node_modules\selenium-standalone\bin\selenium-standalone:105
        throw err;
        ^

Error: Could not download https://selenium-release.storage.googleapis.com/3.0/IEDriverServer_x64_3.0.1.zip
    at Request.<anonymous> (C:\Users\xxxxx\AppData\Roaming\npm\node_modules\selenium-standalone\lib\install.js:273:21)
    at emitOne (events.js:96:13)
    at Request.emit (events.js:188:7)
    at Request.onRequestResponse (C:\Users\xxxxx\AppData\Roaming\npm\node_modules\selenium-standalone\node_modules\request\request.js:986:10)
    at emitOne (events.js:96:13)
    at ClientRequest.emit (events.js:188:7)
    at HTTPParser.parserOnIncomingClient [as onIncoming] (_http_client.js:474:21)
    at HTTPParser.parserOnHeadersComplete (_http_common.js:99:23)
    at TLSSocket.socketOnData (_http_client.js:363:20)
    at emitOne (events.js:96:13)
```

**Comment by stephenash** [comment]
@jlchereau Thanks for the bug report. I have a PR open that fixes this and we should be able to get a fix out soon.

In the meantime, you should be able to use `selenium-standalone install --drivers.ie.version=3.0.0` to get past this issue.

**Comment by stephenash** [comment]
@jlchereau Fixed with the `selenium-standalone@6.0.1` release.

**Comment by jlchereau** [comment]
@stephenash Thx
